### ISSUE TITLE
Balancer "ELI5" educational articles

### ISSUE THREAD

**Issue by jmusighi** [balancer-labs/balancer-bounties#6]
Balancer is one of the more complex protocols in crypto to understand and explain. Content that helps break down these complex concepts into a format that is easy to understand would be a valuable asset.

The project:

A series of clearly written articles, each up to 2,000 words in length. Include visuals where they can add value, such as charts, screenshots, and photos. The articles should be written in an ELI5 manner, so that total beginners can follow along and understand the high level concepts without feeling lost. Balancer involves concepts from multiple disciplines including finance, computer science, math. Assume your audience are not mathematicians, have never studied finance, and have never written a line of code. They are just sophisticated enough to use Metamask or other user-friendly wallets to perform simple DeFi transactions, without necessarily understanding everything that's going on under the hood.

Topics to cover (open to suggestions):

- Balancer overview
- Providing liquidity on Balancer
- Trading on Balancer
- Arbitrage on Balancer
- Building on Balancer
- Smart pool overview
- Managed index funds (use case)
- Smart treasuries (use case)
- Liquidity bootstrapping pools (use case)
- Dynamic fee pools (use case)
- Elastic supply pools (use case)

**Comment by stanislavkozlovski** [comment]
@jmusighi I would be interested in this. I saw @Jonlees1996 claimed this grant earlier - has this been done?

If not, I would like to apply. I am a software engineer by trade and have a good amount of experience writing technical articles in an easy to understand way. Some of my larger pieces are:
1. [Bitcoin Magazine - Bitcoin is the Move](https://bitcoinmagazine.com/articles/bitcoin-is-the-move)
2. [Bitcoin Magazine - How Money Printing Really Works](https://bitcoinmagazine.com/articles/how-money-printing-really-works)
3. [Better Programming - A Thorough Introduction to Distributed Systems](https://medium.com/better-programming/a-thorough-introduction-to-distributed-systems-3b91562c9b3c)
4. [Better Programming - A Thorough Introduction to Apache Kafka](https://medium.com/better-programming/thorough-introduction-to-apache-kafka-6fbf2989bbc1)

I also have some [smaller pieces](https://medium.com/better-programming/kafka-acks-explained-c0515b3b707e) but I thought I'd share the ones I'm most proud of first.

I'm also very intrigued in learning more about Balancer so I think I'll be a good fit to finish this bounty.

**Comment by jmusighi** [comment]
Hey @stanislavkozlovski! Thanks for responding to this bounty. My team is assisting me in reviewing the content you've shared. We'll get back to you asap-- please let me know the best way to reach you.

**Comment by stanislavkozlovski** [comment]
My e-mail is present on my github (stanislav_kozlovski@outlook.com) and I am also part of your Discord - whatever is more convenient to you & your team
